packages/docs/src/lang/en/customization/Accessibility.json

```diff
@@ -20,6 +20,12 @@
   "scopedText4": "However, when using slots there are conditions in which you will need to utilize the passed down properties to get the proper aria setup. This information also applies to `v-autocomplete`, `v-combobox` and `v-overflow-btn` as they all extend the `v-select` component.",
   "scopedText5": "When providing a `v-list-item` component in a slot, the aria attributes are passed through the **attrs** data property and can be bound with `v-bind`.",
   "scopedText6": "In order to properly associate a label with the correct _option_ you will need to bind `attrs.id` to the `aria-labelled-by` on your `v-list-item-title`. This will ensure that the text is properly associated. Keep in mind, this is done automatically for you if you are not using slots.",
+  "screenReaderHeader1": "## Screen Readers",
```

Comment:
I wonder if it should be in `/styles/display` or maybe on both pages?

Also to be consistent with other pages it could be displayed like this:

![image](https://user-images.githubusercontent.com/15625235/80705294-7b105300-8ae6-11ea-9283-cff5ede4294f.png)

Otherwise LGTM

Reply:
Visually looks better now, but I do think that it belongs more to the `/styles/display` page than here. Accessibility page could link to `/styles/display` for "more information about accessibility helper classes" or similar

Reply:
Ah sorry I didn’t read your previous comment correctly. The display page seems to only care about the CSS `display` property for now so adding this would break that pattern, but I’m not against it.
If we were to do that, shall we prefix these classes with `d-`?

- `d-sr-only`
- `d-sr-only-focusable`

Reply:
Hmmm I didn't initially care about showing classes not starting with `d-` on that page but actually I like the idea of `d-` prefix
You can wait for comments from other members to avoid unnecessary work in case most votes will be for no prefix

Reply:
Let's go ahead with the `d-` prefix, it's in line with our display types.

Reply:
@jacekkarczmarczyk @johnleider Added the `d-` prefix and moved the documentation to the Display page 👍